data/en/1_0102.json

```diff
@@ -91,12 +91,13 @@
                         "id": "total-retail-turnover-answer",
                         "alias": "total_turnover",
                         "label": "Total retail turnover",
-                        "description": "Round to the nearest (\u00a3) pound. Do not  include pence",
+                        "description": "",
```

Comment:
`description` is optional, so should be removed rather than set to `""`. This applies to all empty optional description elements in this pull request.

Reply:
All empty description elements removed